### PR TITLE
docs: use `APPDATA` environment variable for windows path

### PR DESCRIPTION
**Pull request by DetachHead**

### CODE REVIEW

**Comment by KotlinIsland** [comment]
I tried this in `powershell-core` 7.2.6 and it said:
```
👉 Set-Location %appdata%\pypoetry
Set-Location: Cannot find path 'C:\%appdata%\pypoetry' because it does not exist.
```

**Comment by DetachHead** [comment]
that's because you're using powershell, but windows explorer uses the `%` syntax in the url bar

**Comment by KotlinIsland** [comment]
image

File: docs/configuration.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ or directly in the `config.toml` file that will be automatically be created when
 This file can typically be found in one of the following directories:
 
 - macOS:   `~/Library/Application Support/pypoetry`
-- Windows: `C:\Users\<username>\AppData\Roaming\pypoetry`
+- Windows: `%appdata%\pypoetry`
```
Comment (neersighted) [inline docs/configuration.md:19]:
```suggestion
- Windows: `%APPDATA%\pypoetry`
```

**Review by neersighted — CHANGES_REQUESTED** [review]

**Review by neersighted — APPROVED** [review]

**Pull request merged**